variables.tf

```diff
@@ -188,6 +188,16 @@ variable "versioning_status" {
   }
 }
 
+variable "retention_status" {
+  description = "A string that indicates the retention status for the log bucket."
+  default     = "Enabled"
+  type        = string
+  validation {
```

Comment:
Thank you for this! We (Truss) should be more thorough with input validation than we have been in the past.